tests/common/dualtor/dual_tor_utils.py

```diff
@@ -777,15 +777,16 @@ def check_nexthops_balance(rand_selected_dut,
         # Hierarchical ECMP validation (in case of standby MUXs):
         # Step 1: Calculate total uplink share.
         total_uplink_share = expect_packet_num * (nexthops_count - len(expected_downlink_ports))
-        # Step 2: Divide uplink share among all uplinks
-        expect_packet_num = total_uplink_share // len(expected_uplink_ports)
+        # Step 2: Divide uplink share among all portchannels
+        expect_packet_num = total_uplink_share // (len(expected_uplink_ports) / 2)
```

Comment:
I have a few comments here:
1. Don't hardcode this value as 2. There are platforms where every potchannel has just one member (TD3) and 2 (TH2). So, please add a logic to find out how many members are present for LAGs, and use that number for division.

2. I think it will be simpler and better if we calculate how many LAGs are there, instead of how many LAG members (expected_uplink_ports). I ask this because we still want `total_uplink_share` packets to be distribute evenly across LAGs. Dividing by LAG members will reduce expectation share per LAG, which is not what we want.

3. Please use `//` instead of `/` for future Python3 compatibility for integer division.